src/SFA.DAS.AANHub.Application/Employers/Commands/CreateEmployerMemberCommandValidator.cs

```diff
@@ -7,7 +7,7 @@ namespace SFA.DAS.AANHub.Application.Employers.Commands
 {
     public class CreateEmployerMemberCommandValidator : AbstractValidator<CreateEmployerMemberCommand>
     {
-        private const string EmployerPairAlreadyCreatedErrorMessage = "UserId and AccountId pair already exist";
+        private const string EmployerPairAlreadyCreatedErrorMessage = "UserRef already exists";
```

Comment:
Variable name can be updated to not have pair any more.

Reply:
Missed that! Updated.